src/containers/app/App.js

```diff
@@ -1,7 +1,15 @@
-import React from 'react';
+import React, { Fragment } from 'react';
+import Producto from '../../components/NuevoProducto';
+import Productos from '../../components/Productos';
 
 function App() {
-  return <h1>Desde App</h1>;
+  return (
+    <Fragment>
```

Comment:
Why are you using Fragment ?

Reply:
I can only return 1 component, for that reason I used Fragment to wrap Product and Products component.